### ISSUE TITLE
Exported windows projects crash immediately after startup

### ISSUE THREAD

**Issue by GoblinOneEye** [dialogic-godot/dialogic#2231]
Hello, I've got the addon working perfectly inside the editor. I can run the project just fine, Dialogic appears to be functioning properly. I've got variables being passed in and out, working dialog trees, characters, etc. Everything appears fine.

However, when I export a windows executable the game crashes immediately after startup. I thought it might be export filters set up improperly, but I believe I've included them as directed:

![image](https://github.com/dialogic-godot/dialogic/assets/169962537/fb4cacd8-bc2f-42fa-b6d4-a24498bab971)

Dialogic is also set up to Autoload:

![image](https://github.com/dialogic-godot/dialogic/assets/169962537/37e85fbb-6725-423b-8901-bee9583819ae)

I tried to run the executable in CMD with --verbose to get a look at what was happening:

![image](https://github.com/dialogic-godot/dialogic/assets/169962537/927647ae-23b0-4bc7-bf39-788514eee96c)
![image](https://github.com/dialogic-godot/dialogic/assets/169962537/307a1a8f-bba1-4d54-9222-370eb65ced47)

I thought maybe I had a problem with my specific project, so I made a new project, reinstalled the addon, and attempted to export a much, much simpler project - to the same results.

Any suggestions? Thank you.

**Comment by coppolaemilio** [comment]
I'm working on a fix for this one 👍

**Comment by GoblinOneEye** [comment]
Thank you, very much appreciated.

**Comment by coppolaemilio** [comment]
Seems to be a regression. Related: https://github.com/godotengine/godot/issues/92021

**Comment by hsandt** [comment]
Copying errors and stacktrace in text to allow text search (I first searched "compilation error" but couldn't find this issue, then had the idea to search "crash" instead).

```
Godot Engine v4.2.1.stable.official.b09f793f5 - https://godotengine.org
libGL error: glx: failed to create dri3 screen
libGL error: failed to load driver: nouveau
OpenGL API 4.6 (Core Profile) Mesa 23.2.1-1ubuntu3.1~22.04.2 - Compatibility - Using Device: Intel - Mesa Intel(R) HD Graphics 4600 (HSW GT2)
 
SCRIPT ERROR: Parse Error: Identifier "EditorInterface" not declared in the current scope.
          at: GDScript::reload (res://addons/dialogic/Core/DialogicUtil.gd:364)
SCRIPT ERROR: Compile Error: 
          at: GDScript::reload (res://addons/dialogic/Resources/character.gd:-1)
SCRIPT ERROR: Compile Error: 
          at: GDScript::reload (res://addons/dialogic/Resources/CharacterResourceLoader.gd:-1)
ERROR: Failed to load script "res://addons/dialogic/Resources/CharacterResourceLoader.gd" with error "Compilation failed".
   at: load (modules/gdscript/gdscript.cpp:2788)
SCRIPT ERROR: Parse Error: Identifier "EditorInterface" not declared in the current scope.
          at: GDScript::reload (res://addons/dialogic/Core/DialogicUtil.gd:364)
SCRIPT ERROR: Compile Error: 
          at: GDScript::reload (res://addons/dialogic/Modules/Background/Transitions/class_dialogic_background_transition.gd:-1)
SCRIPT ERROR: Parse Error: Identifier "EditorInterface" not declared in the current scope.
          at: GDScript::reload (res://addons/dialogic/Core/DialogicUtil.gd:364)
SCRIPT ERROR: Compile Error: 
          at: GDScript::reload (res://addons/dialogic/Resources/dialogic_style.gd:-1)
SCRIPT ERROR: Parse Error: Identifier "EditorInterface" not declared in the current scope.
          at: GDScript::reload (res://addons/dialogic/Core/DialogicUtil.gd:364)
SCRIPT ERROR: Compile Error: 
          at: GDScript::reload (res://addons/dialogic/Modules/Style/subsystem_styles.gd:-1)
SCRIPT ERROR: Parse Error: Identifier "EditorInterface" not declared in the current scope.
          at: GDScript::reload (res://addons/dialogic/Core/DialogicUtil.gd:364)
SCRIPT ERROR: Parse Error: Identifier "EditorInterface" not declared in the current scope.
          at: GDScript::reload (res://addons/dialogic/Core/DialogicUtil.gd:364)
SCRIPT ERROR: Compile Error: 
          at: GDScript::reload (res://addons/dialogic/Modules/Save/subsystem_save.gd:-1)
SCRIPT ERROR: Compile Error: 
          at: GDScript::reload (res://addons/dialogic/Modules/Settings/subsystem_settings.gd:-1)
SCRIPT ERROR: Parse Error: Identifier "EditorInterface" not declared in the current scope.
          at: GDScript::reload (res://addons/dialogic/Core/DialogicUtil.gd:364)
SCRIPT ERROR: Compile Error: 
          at: GDScript::reload (res://addons/dialogic/Modules/Variable/subsystem_variables.gd:-1)
SCRIPT ERROR: Parse Error: Identifier "EditorInterface" not declared in the current scope.
          at: GDScript::reload (res://addons/dialogic/Core/DialogicUtil.gd:364)
SCRIPT ERROR: Compile Error: 
          at: GDScript::reload (res://addons/dialogic/Modules/Glossary/glossary_resource.gd:-1)
SCRIPT ERROR: Compile Error: 
          at: GDScript::reload (res://addons/dialogic/Modules/Glossary/subsystem_glossary.gd:-1)
SCRIPT ERROR: Parse Error: Identifier "EditorInterface" not declared in the current scope.
          at: GDScript::reload (res://addons/dialogic/Core/DialogicUtil.gd:364)
SCRIPT ERROR: Parse Error: Identifier "EditorInterface" not declared in the current scope.
          at: GDScript::reload (res://addons/dialogic/Core/DialogicUtil.gd:364)
SCRIPT ERROR: Parse Error: Identifier "EditorInterface" not declared in the current scope.
          at: GDScript::reload (res://addons/dialogic/Core/DialogicUtil.gd:364)
SCRIPT ERROR: Compile Error: 
          at: GDScript::reload (res://addons/dialogic/Modules/History/subsystem_history.gd:-1)
SCRIPT ERROR: Compile Error: 
          at: GDScript::reload (res://addons/dialogic/Modules/Text/auto_skip.gd:-1)
SCRIPT ERROR: Parse Error: Identifier "EditorInterface" not declared in the current scope.
          at: GDScript::reload (res://addons/dialogic/Core/DialogicUtil.gd:364)
SCRIPT ERROR: Compile Error: 
          at: GDScript::reload (res://addons/dialogic/Modules/Text/manual_advance.gd:-1)
SCRIPT ERROR: Parse Error: Identifier "EditorInterface" not declared in the current scope.
          at: GDScript::reload (res://addons/dialogic/Core/DialogicUtil.gd:364)
SCRIPT ERROR: Compile Error: 
          at: GDScript::reload (res://addons/dialogic/Modules/Core/subsystem_input.gd:-1)
SCRIPT ERROR: Compile Error: 
          at: GDScript::reload (res://addons/dialogic/Modules/Voice/event_voice.gd:-1)
SCRIPT ERROR: Compile Error: 
          at: GDScript::reload (res://addons/dialogic/Modules/Voice/subsystem_voice.gd:-1)
SCRIPT ERROR: Compile Error: 
          at: GDScript::reload (res://addons/dialogic/Modules/Text/auto_advance.gd:-1)
SCRIPT ERROR: Parse Error: Identifier "EditorInterface" not declared in the current scope.
          at: GDScript::reload (res://addons/dialogic/Core/DialogicUtil.gd:364)
SCRIPT ERROR: Compile Error: 
          at: GDScript::reload (res://addons/dialogic/Modules/Text/subsystem_text.gd:-1)
SCRIPT ERROR: Parse Error: Identifier "EditorInterface" not declared in the current scope.
          at: GDScript::reload (res://addons/dialogic/Core/DialogicUtil.gd:364)
SCRIPT ERROR: Compile Error: 
          at: GDScript::reload (res://addons/dialogic/Modules/Character/node_portrait_container.gd:-1)
SCRIPT ERROR: Parse Error: Identifier "EditorInterface" not declared in the current scope.
          at: GDScript::reload (res://addons/dialogic/Core/DialogicUtil.gd:364)
SCRIPT ERROR: Compile Error: 
          at: GDScript::reload (res://addons/dialogic/Modules/Character/subsystem_portraits.gd:-1)
SCRIPT ERROR: Parse Error: Identifier "EditorInterface" not declared in the current scope.
          at: GDScript::reload (res://addons/dialogic/Core/DialogicUtil.gd:364)
SCRIPT ERROR: Compile Error: 
          at: GDScript::reload (res://addons/dialogic/Modules/Choice/subsystem_choices.gd:-1)
SCRIPT ERROR: Parse Error: Identifier "EditorInterface" not declared in the current scope.
          at: GDScript::reload (res://addons/dialogic/Core/DialogicUtil.gd:364)
SCRIPT ERROR: Compile Error: 
          at: GDScript::reload (res://addons/dialogic/Modules/Text/event_text.gd:-1)
SCRIPT ERROR: Compile Error: 
          at: GDScript::reload (res://addons/dialogic/Core/DialogicResourceUtil.gd:-1)
SCRIPT ERROR: Compile Error: 
          at: GDScript::reload (res://addons/dialogic/Modules/Background/subsystem_backgrounds.gd:-1)
SCRIPT ERROR: Parse Error: Identifier "EditorInterface" not declared in the current scope.
          at: GDScript::reload (res://addons/dialogic/Core/DialogicUtil.gd:364)
SCRIPT ERROR: Compile Error: 
          at: GDScript::reload (res://addons/dialogic/Core/DialogicGameHandler.gd:-1)
SCRIPT ERROR: Parse Error: Identifier "EditorInterface" not declared in the current scope.
          at: GDScript::reload (res://addons/dialogic/Core/DialogicUtil.gd:364)
SCRIPT ERROR: Compile Error: 
          at: GDScript::reload (res://addons/dialogic/Resources/event.gd:-1)
SCRIPT ERROR: Compile Error: 
          at: GDScript::reload (res://addons/dialogic/Resources/timeline.gd:-1)
SCRIPT ERROR: Compile Error: 
          at: GDScript::reload (res://addons/dialogic/Resources/TimelineResourceLoader.gd:-1)
ERROR: Failed to load script "res://addons/dialogic/Resources/TimelineResourceLoader.gd" with error "Compilation failed".
   at: load (modules/gdscript/gdscript.cpp:2788)

================================================================
handle_crash: Program crashed with signal 11
Engine version: Godot Engine v4.2.1.stable.official (b09f793f564a6c95dc76acc654b390e68441bd01)
Dumping the backtrace. Please include this when reporting the bug to the project developer.
[1] /lib/x86_64-linux-gnu/libc.so.6(+0x42520) [0x7f61a2d5e520] (??:0)
[2] ./test_with_dialogic_alpha13_enabled.x86_64() [0x2938253] (??:0)
[3] ./test_with_dialogic_alpha13_enabled.x86_64() [0x29a252d] (??:0)
[4] ./test_with_dialogic_alpha13_enabled.x86_64() [0x6774d4] (??:0)
[5] ./test_with_dialogic_alpha13_enabled.x86_64() [0x532f4c] (??:0)
[6] ./test_with_dialogic_alpha13_enabled.x86_64() [0x29df75f] (??:0)
[7] ./test_with_dialogic_alpha13_enabled.x86_64() [0x66cbc3] (??:0)
[8] ./test_with_dialogic_alpha13_enabled.x86_64() [0x532f4c] (??:0)
[9] ./test_with_dialogic_alpha13_enabled.x86_64() [0x29df75f] (??:0)
[10] ./test_with_dialogic_alpha13_enabled.x86_64() [0x67c4aa] (??:0)
[11] ./test_with_dialogic_alpha13_enabled.x86_64() [0x532f4c] (??:0)
[12] ./test_with_dialogic_alpha13_enabled.x86_64() [0x29df75f] (??:0)
[13] ./test_with_dialogic_alpha13_enabled.x86_64() [0x67c4aa] (??:0)
[14] ./test_with_dialogic_alpha13_enabled.x86_64() [0x569611] (??:0)
[15] ./test_with_dialogic_alpha13_enabled.x86_64() [0x106abfa] (??:0)
[16] ./test_with_dialogic_alpha13_enabled.x86_64() [0x2b3f5e0] (??:0)
[17] ./test_with_dialogic_alpha13_enabled.x86_64() [0x103b936] (??:0)
[18] ./test_with_dialogic_alpha13_enabled.x86_64() [0x103b8e4] (??:0)
[19] ./test_with_dialogic_alpha13_enabled.x86_64() [0x107f35f] (??:0)
[20] ./test_with_dialogic_alpha13_enabled.x86_64() [0x41a6e1] (??:0)
[21] /lib/x86_64-linux-gnu/libc.so.6(+0x29d90) [0x7f61a2d45d90] (??:0)
[22] /lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0x80) [0x7f61a2d45e40] (??:0)
[23] ./test_with_dialogic_alpha13_enabled.x86_64() [0x4218fa] (??:0)
-- END OF BACKTRACE --
================================================================
```

**Comment by CakeVR** [comment]
> Copying errors and stacktrace in text to allow text search (I first searched "compilation error" but couldn't find this issue, then had the idea to search "crash" instead).
> 
> ```
> Godot Engine v4.2.1.stable.official.b09f793f5 - https://godotengine.org
> libGL error: glx: failed to create dri3 screen
> libGL error: failed to load driver: nouveau
> OpenGL API 4.6 (Core Profile) Mesa 23.2.1-1ubuntu3.1~22.04.2 - Compatibility - Using Device: Intel - Mesa Intel(R) HD Graphics 4600 (HSW GT2)
>  
> SCRIPT ERROR: Parse Error: Identifier "EditorInterface" not declared in the current scope.
>           at: GDScript::reload (res://addons/dialogic/Core/DialogicUtil.gd:364)
> SCRIPT ERROR: Compile Error: 
>           at: GDScript::reload (res://addons/dialogic/Resources/character.gd:-1)
> SCRIPT ERROR: Compile Error: 
>           at: GDScript::reload (res://addons/dialogic/Resources/CharacterResourceLoader.gd:-1)
> ERROR: Failed to load script "res://addons/dialogic/Resources/CharacterResourceLoader.gd" with error "Compilation failed".
>    at: load (modules/gdscript/gdscript.cpp:2788)
> SCRIPT ERROR: Parse Error: Identifier "EditorInterface" not declared in the current scope.
>           at: GDScript::reload (res://addons/dialogic/Core/DialogicUtil.gd:364)
> SCRIPT ERROR: Compile Error: 
>           at: GDScript::reload (res://addons/dialogic/Modules/Background/Transitions/class_dialogic_background_transition.gd:-1)
> SCRIPT ERROR: Parse Error: Identifier "EditorInterface" not declared in the current scope.
>           at: GDScript::reload (res://addons/dialogic/Core/DialogicUtil.gd:364)
> SCRIPT ERROR: Compile Error: 
>           at: GDScript::reload (res://addons/dialogic/Resources/dialogic_style.gd:-1)
> SCRIPT ERROR: Parse Error: Identifier "EditorInterface" not declared in the current scope.
>           at: GDScript::reload (res://addons/dialogic/Core/DialogicUtil.gd:364)
> SCRIPT ERROR: Compile Error: 
>           at: GDScript::reload (res://addons/dialogic/Modules/Style/subsystem_styles.gd:-1)
> SCRIPT ERROR: Parse Error: Identifier "EditorInterface" not declared in the current scope.
>           at: GDScript::reload (res://addons/dialogic/Core/DialogicUtil.gd:364)
> SCRIPT ERROR: Parse Error: Identifier "EditorInterface" not declared in the current scope.
>           at: GDScript::reload (res://addons/dialogic/Core/DialogicUtil.gd:364)
> SCRIPT ERROR: Compile Error: 
>           at: GDScript::reload (res://addons/dialogic/Modules/Save/subsystem_save.gd:-1)
> SCRIPT ERROR: Compile Error: 
>           at: GDScript::reload (res://addons/dialogic/Modules/Settings/subsystem_settings.gd:-1)
> SCRIPT ERROR: Parse Error: Identifier "EditorInterface" not declared in the current scope.
>           at: GDScript::reload (res://addons/dialogic/Core/DialogicUtil.gd:364)
> SCRIPT ERROR: Compile Error: 
>           at: GDScript::reload (res://addons/dialogic/Modules/Variable/subsystem_variables.gd:-1)
> SCRIPT ERROR: Parse Error: Identifier "EditorInterface" not declared in the current scope.
>           at: GDScript::reload (res://addons/dialogic/Core/DialogicUtil.gd:364)
> SCRIPT ERROR: Compile Error: 
>           at: GDScript::reload (res://addons/dialogic/Modules/Glossary/glossary_resource.gd:-1)
> SCRIPT ERROR: Compile Error: 
>           at: GDScript::reload (res://addons/dialogic/Modules/Glossary/subsystem_glossary.gd:-1)
> SCRIPT ERROR: Parse Error: Identifier "EditorInterface" not declared in the current scope.
>           at: GDScript::reload (res://addons/dialogic/Core/DialogicUtil.gd:364)
> SCRIPT ERROR: Parse Error: Identifier "EditorInterface" not declared in the current scope.
>           at: GDScript::reload (res://addons/dialogic/Core/DialogicUtil.gd:364)
> SCRIPT ERROR: Parse Error: Identifier "EditorInterface" not declared in the current scope.
>           at: GDScript::reload (res://addons/dialogic/Core/DialogicUtil.gd:364)
> SCRIPT ERROR: Compile Error: 
>           at: GDScript::reload (res://addons/dialogic/Modules/History/subsystem_history.gd:-1)
> SCRIPT ERROR: Compile Error: 
>           at: GDScript::reload (res://addons/dialogic/Modules/Text/auto_skip.gd:-1)
> SCRIPT ERROR: Parse Error: Identifier "EditorInterface" not declared in the current scope.
>           at: GDScript::reload (res://addons/dialogic/Core/DialogicUtil.gd:364)
> SCRIPT ERROR: Compile Error: 
>           at: GDScript::reload (res://addons/dialogic/Modules/Text/manual_advance.gd:-1)
> SCRIPT ERROR: Parse Error: Identifier "EditorInterface" not declared in the current scope.
>           at: GDScript::reload (res://addons/dialogic/Core/DialogicUtil.gd:364)
> SCRIPT ERROR: Compile Error: 
>           at: GDScript::reload (res://addons/dialogic/Modules/Core/subsystem_input.gd:-1)
> SCRIPT ERROR: Compile Error: 
>           at: GDScript::reload (res://addons/dialogic/Modules/Voice/event_voice.gd:-1)
> SCRIPT ERROR: Compile Error: 
>           at: GDScript::reload (res://addons/dialogic/Modules/Voice/subsystem_voice.gd:-1)
> SCRIPT ERROR: Compile Error: 
>           at: GDScript::reload (res://addons/dialogic/Modules/Text/auto_advance.gd:-1)
> SCRIPT ERROR: Parse Error: Identifier "EditorInterface" not declared in the current scope.
>           at: GDScript::reload (res://addons/dialogic/Core/DialogicUtil.gd:364)
> SCRIPT ERROR: Compile Error: 
>           at: GDScript::reload (res://addons/dialogic/Modules/Text/subsystem_text.gd:-1)
> SCRIPT ERROR: Parse Error: Identifier "EditorInterface" not declared in the current scope.
>           at: GDScript::reload (res://addons/dialogic/Core/DialogicUtil.gd:364)
> SCRIPT ERROR: Compile Error: 
>           at: GDScript::reload (res://addons/dialogic/Modules/Character/node_portrait_container.gd:-1)
> SCRIPT ERROR: Parse Error: Identifier "EditorInterface" not declared in the current scope.
>           at: GDScript::reload (res://addons/dialogic/Core/DialogicUtil.gd:364)
> SCRIPT ERROR: Compile Error: 
>           at: GDScript::reload (res://addons/dialogic/Modules/Character/subsystem_portraits.gd:-1)
> SCRIPT ERROR: Parse Error: Identifier "EditorInterface" not declared in the current scope.
>           at: GDScript::reload (res://addons/dialogic/Core/DialogicUtil.gd:364)
> SCRIPT ERROR: Compile Error: 
>           at: GDScript::reload (res://addons/dialogic/Modules/Choice/subsystem_choices.gd:-1)
> SCRIPT ERROR: Parse Error: Identifier "EditorInterface" not declared in the current scope.
>           at: GDScript::reload (res://addons/dialogic/Core/DialogicUtil.gd:364)
> SCRIPT ERROR: Compile Error: 
>           at: GDScript::reload (res://addons/dialogic/Modules/Text/event_text.gd:-1)
> SCRIPT ERROR: Compile Error: 
>           at: GDScript::reload (res://addons/dialogic/Core/DialogicResourceUtil.gd:-1)
> SCRIPT ERROR: Compile Error: 
>           at: GDScript::reload (res://addons/dialogic/Modules/Background/subsystem_backgrounds.gd:-1)
> SCRIPT ERROR: Parse Error: Identifier "EditorInterface" not declared in the current scope.
>           at: GDScript::reload (res://addons/dialogic/Core/DialogicUtil.gd:364)
> SCRIPT ERROR: Compile Error: 
>           at: GDScript::reload (res://addons/dialogic/Core/DialogicGameHandler.gd:-1)
> SCRIPT ERROR: Parse Error: Identifier "EditorInterface" not declared in the current scope.
>           at: GDScript::reload (res://addons/dialogic/Core/DialogicUtil.gd:364)
> SCRIPT ERROR: Compile Error: 
>           at: GDScript::reload (res://addons/dialogic/Resources/event.gd:-1)
> SCRIPT ERROR: Compile Error: 
>           at: GDScript::reload (res://addons/dialogic/Resources/timeline.gd:-1)
> SCRIPT ERROR: Compile Error: 
>           at: GDScript::reload (res://addons/dialogic/Resources/TimelineResourceLoader.gd:-1)
> ERROR: Failed to load script "res://addons/dialogic/Resources/TimelineResourceLoader.gd" with error "Compilation failed".
>    at: load (modules/gdscript/gdscript.cpp:2788)
> 
> ================================================================
> handle_crash: Program crashed with signal 11
> Engine version: Godot Engine v4.2.1.stable.official (b09f793f564a6c95dc76acc654b390e68441bd01)
> Dumping the backtrace. Please include this when reporting the bug to the project developer.
> [1] /lib/x86_64-linux-gnu/libc.so.6(+0x42520) [0x7f61a2d5e520] (??:0)
> [2] ./test_with_dialogic_alpha13_enabled.x86_64() [0x2938253] (??:0)
> [3] ./test_with_dialogic_alpha13_enabled.x86_64() [0x29a252d] (??:0)
> [4] ./test_with_dialogic_alpha13_enabled.x86_64() [0x6774d4] (??:0)
> [5] ./test_with_dialogic_alpha13_enabled.x86_64() [0x532f4c] (??:0)
> [6] ./test_with_dialogic_alpha13_enabled.x86_64() [0x29df75f] (??:0)
> [7] ./test_with_dialogic_alpha13_enabled.x86_64() [0x66cbc3] (??:0)
> [8] ./test_with_dialogic_alpha13_enabled.x86_64() [0x532f4c] (??:0)
> [9] ./test_with_dialogic_alpha13_enabled.x86_64() [0x29df75f] (??:0)
> [10] ./test_with_dialogic_alpha13_enabled.x86_64() [0x67c4aa] (??:0)
> [11] ./test_with_dialogic_alpha13_enabled.x86_64() [0x532f4c] (??:0)
> [12] ./test_with_dialogic_alpha13_enabled.x86_64() [0x29df75f] (??:0)
> [13] ./test_with_dialogic_alpha13_enabled.x86_64() [0x67c4aa] (??:0)
> [14] ./test_with_dialogic_alpha13_enabled.x86_64() [0x569611] (??:0)
> [15] ./test_with_dialogic_alpha13_enabled.x86_64() [0x106abfa] (??:0)
> [16] ./test_with_dialogic_alpha13_enabled.x86_64() [0x2b3f5e0] (??:0)
> [17] ./test_with_dialogic_alpha13_enabled.x86_64() [0x103b936] (??:0)
> [18] ./test_with_dialogic_alpha13_enabled.x86_64() [0x103b8e4] (??:0)
> [19] ./test_with_dialogic_alpha13_enabled.x86_64() [0x107f35f] (??:0)
> [20] ./test_with_dialogic_alpha13_enabled.x86_64() [0x41a6e1] (??:0)
> [21] /lib/x86_64-linux-gnu/libc.so.6(+0x29d90) [0x7f61a2d45d90] (??:0)
> [22] /lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0x80) [0x7f61a2d45e40] (??:0)
> [23] ./test_with_dialogic_alpha13_enabled.x86_64() [0x4218fa] (??:0)
> -- END OF BACKTRACE --
> ================================================================
> ```

This is unrelated, please update to Alpha 14.

**Comment by hsandt** [comment]
> This is unrelated, please update to Alpha 14.

Thanks, upgrading to Alpha 14 fixed both the "Compilation failed" errors in terminal and the crash on start. I can now play, building from Godot 4.2.1 / 4.2.2!

... but then that means it was not related to Godot issue 92021 (which is only fixed on bleeding edge, even after Godot 4.3.dev6), but the same as the OP's issue (I got the same errors looking at their screenshots, and they while didn't put their Dialogic version but they posted it 2 weeks ago, so before Alpha 14 related 5 days ago).

Or were you referring to @coppolaemilio 's post, meaning "This is unrelated to that Godot issue" ? Maybe you misclicked on my post instead when doing Quote reply?

Either way, this saves the day. I can finish my jam now. It's such a big fix it could be added in the [Alpha 14 changelog](https://github.com/dialogic-godot/dialogic/releases/tag/2.0-alpha-14)!
